cracking-the-coding-interview/problems/2.linked_lists/1.removeDups.ts

```diff
@@ -0,0 +1,9 @@
+// 1. * Remove Dups *: Write code to remove duplicates from an unsorted linked list.FOLLOW UP
+// How would you solve this problem if a temporary buffer is not allowed ?
+
+const removeDups = (list) => {
+  const setItems = new Set(list);
+  return [...setItems.values()]
```

Comment:
The input is supposed to be a linked list. Also, the extra prompt is about not using a set.